docs/specs/images.md

```diff
@@ -0,0 +1,111 @@
+# Authgear Images
+
+Authgear Images server provides images storing and resizing for user profile attributes.
+
+## Endpoints
+
+### Serving image
+
+The serving image endpoint obtains the images from the object store, performs resizing, and serves the image directly.
+
+The serving image endpoint is publicly accessible.
+
+```
+GET /_images/APP_ID/OBJECT_ID/OPTIONS HTTP/1.1
+```
+
+- `APP_ID`: The authgear app id
+- `OBJECT_ID`: The file object id
+- `OPTIONS`: Following options are supported.
+    - Resizing options, e.g. `w=400,h=300`
```

Comment:
We should document the options we support.

https://developers.cloudflare.com/images/image-resizing/url-format/

We should at least support `w,h,fit,q`

Ideally, we should process and discard essential EXIF data like orientation so that less smart client will not be confused by EXIF orientation.